RoFILib/generate/main.cpp

```diff
@@ -102,17 +103,17 @@ ConfigPair generateTest(unsigned modules, unsigned path, unsigned step, unsigned
     {
         std::vector<Action::Rotate> rotate;
         std::vector<Action::Reconnect> reconnect;
-        goal.generateReconnect(reconnect);
-        goal.generateRotations(rotate, step);
+        generateReconnect(goal, reconnect);
+        generateRotations(goal, rotate, step);
         std::uniform_int_distribution<unsigned long> flip(0, 5);
         //if ((flip(e) > 1))
```

Comment:
What is the reason for commented-out if?

Reply:
No idea

Reply:
Could you look at it? I don't want to keep commented conditions in the code.